packages/client/src/StreamrClient.ts

```diff
@@ -582,6 +582,15 @@ export class StreamrClient {
         await Promise.all(tasks)
     })
 
+    /**
+     * Get diagnostic info about the underlying network. Useful for debugging issues.
+     *
+     * @remark returned object's structure can change without semver considerations
+     */
+    async getDiagnosticInfo(): Promise<Record<string, unknown>> {
```

Comment:
The method names could match: maybe `getDiagnosticInfo` on both?

Reply:
Yeah mistake, fixed.